stdlib/public/SwiftShims/RuntimeShims.h

```diff
@@ -48,6 +48,9 @@ const char *_swift_stdlib_strtof_clocale(const char *nptr, float *outResult);
 SWIFT_RUNTIME_STDLIB_API
 void _swift_stdlib_immortalize(void *obj);
   
+SWIFT_RUNTIME_STDLIB_API
+void _swift_stdlib_set_objc_complications_forbidden(void *obj, int forbid);
```

Comment:
Do we have a way to mark availability of runtime entry points?

Reply:
I have no idea! I hope so

Reply:
Couldn't you do something like:
```swift
@_silgen_name("_swift_stdlib_set_objc_complications_forbidden")
@available(/*availability stuff here*/)
internal func _set_objc_complications_forbidden(_ addr: UnsafeMutableRawPointer, _ forbid: Int)
```
to give this function availability? (You can rename function name and parameters and access control, just giving an example).